platform-core-module/src/main/java/com/softicar/platform/core/module/page/service/login/PageServiceLoginDiv.java

```diff
@@ -155,7 +157,17 @@ public MaintenanceElement() {
 
 				setCssClass(PageCssClasses.PAGE_SERVICE_LOGIN_MAINTENANCE_DIV);
 				if (AGMaintenanceWindow.isMaintenanceInProgress()) {
-					appendChild(new DomMessageDiv(DomMessageType.WARNING, CoreI18n.MAINTENANCE_IS_CURRENTLY_IN_PROGRESS));
+					DayTime expectedEnd = AGMaintenanceWindow.TABLE//
```

Comment:
It would be more elegant, to call some function `AGMaintenanceWindow.getMaintenanceInProgress()` returning an `Optional<AGMaintenanceWindow>` and then working with it. I think I already suggested this some PRs ago. ;)